### ISSUE TITLE
Azure File Copy task creates anonymously accessible containers by default

### ISSUE THREAD

**Issue by tstoian** [microsoft/azure-pipelines-tasks#13196]
## Note  
Issues in this repo are for tracking bugs, feature requests and questions for the tasks in this repo 

For a list:  
https://github.com/Microsoft/azure-pipelines-tasks/tree/master/Tasks

If you have an issue or request for the Azure Pipelines service, use developer community instead:  

https://developercommunity.visualstudio.com/spaces/21/index.html )

## Required Information

Entering this information will route you directly to the right team and expedite traction.

**Question, Bug, or Feature?**  
*Type*: Question

**Enter Task Name**: Azure File Copy  

list here (V# not needed):  
https://github.com/Microsoft/azure-pipelines-tasks/tree/master/Tasks

## Environment
- Server - Azure Pipelines or TFS on-premises?
    
    - If using TFS on-premises, provide the version: 
    
    - If using Azure Pipelines, provide the account name, team project name, build definition name/build number: 

- Agent - Hosted or Private: 
Hosted
Image: vs2017-win2016
Agent: Azure Pipelines 5

## Issue Description

I recently upgraded a task the Azure File Copy task from v1 to v4 and noticed a change in behavior. I'm using the task to copy files to an Azure Storage Account. The container that gets created has its 'Public Access Level' attribute set to 'Container' whereas the older version was setting it to 'Private'. The consequence is that anyone having (or guessing) the container URI gets access to the container. Since the task also generates a time-constrained SAS Token, I doubt that anonymous access to the container is really intended.

Digging a little bit in the code, I noticed that the Create-AzureContainer function will set the permission to 'Container' is the storage account is not premium. If it is premium, the default permission ('Private') is used. I don't see a compelling reason to tie security option to the performance characteristics of the Storage Account.
https://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/AzureFileCopyV4/AzureUtilityAz1.0.ps1, lines 57-59

To sum it up, my questions are: is it intended to create the blob container with anonymous access by default or is this a bug? Is there a way to change this behavior and the create private containers instead?

### Task logs

[Enable debug logging and please provide the zip file containing all the logs for a speedy resolution]

## Troubleshooting
Checkout how to troubleshoot failures and collect debug logs: https://docs.microsoft.com/en-us/vsts/build-release/actions/troubleshooting

### Error logs

[Insert error from the logs here for a quick overview]

**Comment by tstoian** [comment]
Any updates on the question? If the public access to the container is not intended, this might be a serious security issue.

**Comment by 20shivangi** [comment]
@tstoian We are able to reproduce the issue, we will work on this

**Comment by 20shivangi** [comment]
@tstoian Able to repro this. Thanks for reporting the issue.

**Comment by 20shivangi** [comment]
@tstoian The issue has been fixed in PR : https://github.com/microsoft/azure-pipelines-tasks/pull/13792
The fix will get deployed within 2-3 weeks. Please wait till that time
